src/wsk-tasks/babel.js

```diff
@@ -20,25 +20,58 @@
 'use strict';
 
 const gulp = require('gulp');
-const babel = require('gulp-babel');
 const uglify = require('gulp-uglify');
-const rollup = require('gulp-rollup');
 const sourcemaps = require('gulp-sourcemaps');
+const browserify = require('browserify');
+const buffer = require('vinyl-buffer');
+const source = require('vinyl-source-stream');
+const path = require('path');
+const glob = require('glob');
+
+function bundleJS(fullFilePath) {
+  return new Promise((resolve, reject) => {
+    const browserifyBundles = browserify({
+      entries: fullFilePath
+    });
+
+    // Rollupify reduces the size of the final output but increases build
+    // time to do it so enable for production build only
+    if (GLOBAL.config.env === 'prod') {
+      browserifyBundles.transform('rollupify');
+    }
+
+    let stream = browserifyBundles
+    .transform('babelify', {presets: ['es2015']})
+    .bundle()
+    // `source` Converts Browserify's Node Stream to a Gulp Stream
+    // Use path.relative to make the file have the correct home in `dest`
+    .pipe(
+      source(path.join('.', path.relative(GLOBAL.config.src, fullFilePath)))
+    )
+    .pipe(buffer())
+    .pipe(sourcemaps.init());
+
+    if (GLOBAL.config.env === 'prod') {
+      stream = stream.pipe(uglify());
+    }
+
+    stream.pipe(sourcemaps.write('.'))
+    .pipe(gulp.dest(GLOBAL.config.dest))
+    .on('error', reject)
+    .on('end', () => resolve());
+  });
```

Comment:
I have a similar question. It's unclear why we aren't just returning the stream here. I think we should try to avoid making assumptions about other functions needing the stream wrapped up and they can take care of that themselves.